Core/Newsletter2Go/Controllers/Api/NewsletterCustomers.php

```diff
@@ -90,18 +97,18 @@ private function getCustomers()
      *
      * @return array
      */
-    private function getOnlySubscribers($group, $emails = array())
+    private function getOnlySubscribers($group, array $emails = [])
     {
         $q = 'SELECT ma.email '
-            . 'FROM s_campaigns_mailaddresses ma ';
+            . 'FROM s_campaigns_mailaddresses ma '
+            . 'WHERE ma.email NOT IN (SELECT email FROM s_user) ';
```

Comment:
This new query condition came from the restored code... I'm not sure if it is correct.